exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/ApiController.java

```diff
@@ -160,17 +174,50 @@ private void getCounter(RoutingContext rc) {
     }
   }
 
+  private void getHeight(RoutingContext rc) {
+    try {
+      Height height = service.getHeight();
+      Gson gson = QaTransactionGson.instance();
+      rc.response()
+          .putHeader("Content-Type", "application/json")
+          .end(gson.toJson(height));
+    } catch (RuntimeException ex) {
```

Comment:
Whats the purpose of this try {} catch () block ?

Reply:
because `com.exonum.binding.blockchain.Blockchain#getHeight` may thow `RuntimeException`. From the javadoc:

> @throws RuntimeException if the "genesis block" was not created